API/CLI.md

```diff
@@ -15,6 +15,12 @@ The best place to go next is [the Tessel 2 start experience](http://tessel.io/t2
 ## Updating Tessel 2 On-board OS/Firmware
 
 ### How do I know if I need to update my T2?
+If the Tessel Module is connected to the Internet, then the CLI informs the user, which firmware version the module is currently working on. Along with this, it would also let the user know about the latest firmware version available and the command using which the user may get the newer version.
+
+This can be seen in the image below:
+
+![CLI giving update Information](https://github.com/276linesofCode/t2-docs/blob/master/images/t2-update.png)
```

Comment:
This image shows the CLI checking whether T2's CLI is up to date, not whether the connected Tessel's firmware is up to date.